scripts/linux_share_build.sh

```diff
@@ -3,33 +3,82 @@ PROC=$((`grep -c processor /proc/cpuinfo`))
 
 function install_prerequisites() {
     TZ=America/Chicago
-    ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
-    apt-get update -y
-    apt-get install -y g++ libeigen3-dev libhdf5-dev patchelf git cmake
+    $SUDO ln -snf /usr/share/zoneinfo/$TZ /etc/localtime
```

Comment:
We should be able to install the `sudo` command in the docker images, right?

Reply:
yes we could I was just scared of the potential effect of using sudo there, as those script are also there for direct use...

Reply:
Ah yeah that's a good point. Nevermind.